src/Kdyby/Doctrine/Mapping/ClassMetadataFactory.php

```diff
@@ -70,10 +70,7 @@ protected function loadMetadata($name)
 			throw new Kdyby\Doctrine\MissingClassException("Metadata of class $name was not found, because the class is missing or cannot be autoloaded.");
 		}
 
-		$result = parent::loadMetadata($name);
-		if ($name !== $origName) {
-			$this->setMetadataFor($origName, $this->getMetadataFor($name));
-		}
+		$result = parent::loadMetadata($origName);
```

Comment:
The variable is no longer needed, just use `return parent::loadMetadata($origName);`.